src/index.js

```diff
@@ -423,6 +430,47 @@ function setCookiesAndTokens(tokens) {
   setCookie(tokens.id.value, tokens.id.cookieOptions, "id");
   setCookie(tokens.refresh.value, tokens.refresh.cookieOptions, "refresh");
   setTokensFromCookies();
+  setUser();
+}
+
+/**
+ * Define user attributes & methods based on access & ID token.
+ */
+function setUser() {
+  if (!store.idToken) {
+    throw new Error("ID token has not been set.");
+  }
+  if (!store.accessToken) {
+    throw new Error("Access token has not been set.");
+  }
+
+  const decodedIdToken = jwt.decode(store.idToken);
+  const decodedAccessToken = jwt.decode(store.accessToken);
+
+  store.user = {
+    ...user({ idToken: decodedIdToken, accessToken: decodedAccessToken }),
+    ...apiResource({
+      store,
+      path: "/users",
+      id: decodedAccessToken.userId,
+      afterUpdate: refresh,
+    }),
+  };
+}
+
+async function refresh() {
+  const { data } = await axios.get({
+    url: `${apiUrl}tenants/${store.tenantId}/refresh`,
+    headers: {
+      authorization: `Bearer ${store.accessToken}`,
+    },
+  });
+
+  if (data.tokens) {
+    setCookiesAndTokens(data.tokens);
+  } else {
+    throw new Error("Problem refreshing tokens.");
+  }
```

Comment:
This can be modified/thrown away. Is only here to satisfy the current tests (`setCookiesAndTokens` needs to be called in order to update the tokens).